app/services/recent-events.ts

```diff
@@ -52,6 +57,59 @@ const subscriptionMap = (subPlan: string) => {
   }[subPlan];
 };
 
+function getHashForRecentEvent(event: IRecentEvent) {
```

Comment:
Should we leave some sort of comment that this is duplicated from the website, and that this will be refactored eventually?

Reply:
Good idea, will add a comment